src/app/store.ts

```diff
@@ -1,10 +1,21 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The function `combineSlices` does not exist in Redux Toolkit. It should be `combineReducers` to correctly combine the reducers.